changelogs/unreleased/000-msdolbey

```diff
@@ -0,0 +1,3 @@
+## All changes
+
+- made some markdown changes in the README (#xx, @msdolbey)
```

Comment:
```suggestion
- made some markdown changes in the README (#106, @msdolbey)
```

Reply:
Thanks.
Plugins don't use these changelog files to generate the release note, so I think it's OK to not change. I will manually create the release note for GA.